module/zfs/vdev.c

```diff
@@ -4437,6 +4437,37 @@ vdev_get_stats_ex_impl(vdev_t *vd, vdev_stat_t *vs, vdev_stat_ex_t *vsx)
 	}
 }
 
+/*
+ * Get mirror slog alloc
+ */
+static uint64_t
+mirror_slog_usage(vdev_t* vd, vdev_stat_t* vs)
+{
+    /*
+    * If we're getting stats on the root vdev, aggregate the I/O counts
+    * over all top-level vdevs (i.e. the direct children of the root).
+    */
+    if (!vd->vdev_ops->vdev_op_leaf) {
+	if (vs) {
+	    memset(vs->vs_ops, 0, sizeof(vs->vs_ops));
+	    memset(vs->vs_bytes, 0, sizeof(vs->vs_bytes));
+	}
+
+	for (int c = 0; c < vd->vdev_children; c++) {
+	    vdev_t* cvd = vd->vdev_child[c];
+	    vdev_stat_t* cvs = &cvd->vdev_stat;
+	    vdev_stat_ex_t* cvsx = &cvd->vdev_stat_ex;
+
+	    mirror_slog_usage(cvd, cvs);
+	    if (cvd->vdev_islog)
+		return cvs->vs_alloc;
+	    if (vs)
+		vdev_get_child_stat(cvd, vs, cvs);
```

Comment:
Do we need this function. we are not using those values which they fill in vdev_get_child_stat ? . we are using only cvs->vs_alloc ..